pkg/subscriber/processor/metrics.go

```diff
@@ -79,6 +80,14 @@ var (
 			Help:      "Histogram of message handling duration (seconds)",
 			Buckets:   prometheus.DefBuckets,
 		}, []string{"subscriber", "code"})
+
+	evaluationEventCounter = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Namespace: "bucketeer",
+			Subsystem: "subscriber",
+			Name:      "evaluation_event_total",
+			Help:      "Total number of evaluation events",
+		}, []string{"tag", "feature_id", "app_version"})
```

Comment:
How about also adding the sdk_version? It could be helpful.

```suggestion
		}, []string{"sdk_version", "tag", "feature_id", "app_version"})
```

Reply:
Sounds good!

Reply:
Fixed 👍